Makefile

```diff
@@ -1,51 +1,23 @@
-.PHONY: all clean install build
-all: build doc
 
-NAME=mirage-block-xen
-IMAGE=$(NAME)
-J=4
+.PHONY: build clean test
 
-include config.mk
-config.mk: configure
-	./configure
+build:
+	jbuilder build @install --dev
 
-configure: configure.ml
-	ocamlfind ocamlopt -package "cmdliner,findlib" -linkpkg $< -o $@
+test:
+	jbuilder runtest
```

Comment:
you might want to do `jbuilder runtest --dev` to fix warning in tests (this already spotted a errors for me in the past, including unused tests which I have forgotten to run...)